### PR TITLE
RUN-437: Add OpenAPI Spec annotations

### PR DESCRIPTION
**Pull request by gschueler**
**Is this a bugfix, or an enhancement? Please describe.**
Enhancement: Add missing OpenAPI Spec annotations

### CODE REVIEW

File: rundeckapp/grails-app/controllers/rundeck/controllers/MenuController.groovy [hunk 1]
```diff
@@ -2641,6 +2826,32 @@ class MenuController extends ControllerBase implements ApplicationContextAware{
...
+Authorization required: `read` or `view` for the job.
+
+Since: V18''',
+        tags=['jobs'],
+        parameters = @Parameter(
+            name = "id",
+            description = "Job ID",
+            in = ParameterIn.PATH,
+            required = true,
+            content = @Content(schema = @Schema(implementation = String))
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/MenuController.groovy:2844]: why this is not defined as `schema = @Schema(type = 'string')`  as the other parameters?

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 2]
```diff
@@ -606,6 +654,36 @@ class ProjectController extends ControllerBase{
...
+    @Tags(
+            [
+                    @Tag(name="project")
+            ]
+    )
+    @ApiResponse(
+            responseCode = "200",
+            description = "Project details",
+            content = @Content(
+                    mediaType = "application/json",
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:683]: I checked the result file using `autorest` (what we used for generate the `go-rundeck` client) and complained the `schema` is missing here

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 3]
```diff
@@ -840,6 +989,47 @@ class ProjectController extends ControllerBase{
...
+                    @Tag(name="project")
+            ]
+    )
+    @ApiResponse(
+            responseCode = "200",
+            description = "Project details",
+            content = [
+                    @Content(
+                            mediaType = "application/text",
+                            examples = @ExampleObject('''key=value
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1021]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 4]
```diff
@@ -840,6 +989,47 @@ class ProjectController extends ControllerBase{
...
+key2=value''')
+                    ),
+                    @Content(
+                            mediaType = "application/json",
+                            examples = @ExampleObject('''{
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1026]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 5]
```diff
@@ -874,9 +1064,368 @@ class ProjectController extends ControllerBase{
...
+            [
+                    @Tag(name="project")
+            ]
+    )
+    @ApiResponse(
+            responseCode = "200",
+            description = "Project details",
+            content = [
+                    @Content(
+                            mediaType = "application/text",
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1109]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 6]
```diff
@@ -874,9 +1064,368 @@ class ProjectController extends ControllerBase{
...
+  application: rundeck
+for:
+  project:
+    - allow: read
+by:
+  group: build''')
+                    ),
+                    @Content(
+                            mediaType = "application/yaml",
+                            examples = @ExampleObject('''description: "my policy"
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1121]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 7]
```diff
@@ -874,9 +1064,368 @@ class ProjectController extends ControllerBase{
...
+  application: rundeck
+for:
+  project:
+    - allow: read
+by:
+  group: build''')
+                            ),
+                            @Content(
+                                    mediaType = "application/yaml",
+                                    examples = @ExampleObject('''description: "my policy"
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1174]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 8]
```diff
@@ -874,9 +1064,368 @@ class ProjectController extends ControllerBase{
...
+  application: rundeck
+for:
+  project:
+    - allow: read
+by:
+  group: build''')
+                            ),
+                            @Content(
+                                    mediaType = "application/json",
+                                    examples = @ExampleObject('''{
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1185]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 9]
```diff
@@ -1144,6 +1693,77 @@ class ProjectController extends ControllerBase{
...
+                    @Tag(name="project")
+            ]
+    )
+    @ApiResponse(
+            responseCode = "200",
+            description = "Project details",
+            content = [
+                    @Content(
+                            mediaType = "application/text",
+                            examples = @ExampleObject('''The readme contents''')
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1751]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 10]
```diff
@@ -1144,6 +1693,77 @@ class ProjectController extends ControllerBase{
...
+                    ),
+                    @Content(
+                            mediaType = "application/json",
+                            examples = @ExampleObject('''{"contents":"The readme contents"}''')
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1755]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 11]
```diff
@@ -1237,6 +1857,61 @@ class ProjectController extends ControllerBase{
...
+                    @Tag(name="project")
+            ]
+    )
+    @ApiResponse(
+            responseCode = "200",
+            description = "Project details",
+            content = [
+                    @Content(
+                            mediaType = "application/text",
+                            examples = @ExampleObject('''The readme contents''')
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1899]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 12]
```diff
@@ -1237,6 +1857,61 @@ class ProjectController extends ControllerBase{
...
+                    ),
+                    @Content(
+                            mediaType = "application/json",
+                            examples = @ExampleObject('''{"contents":"The readme contents"}''')
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1903]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 13]
```diff
@@ -1305,6 +2024,81 @@ class ProjectController extends ControllerBase{
...
+                    @Tag(name="project")
+            ]
+    )
+    @ApiResponse(
+            responseCode = "200",
+            description = "Project details",
+            content = [
+                    @Content(
+                            mediaType = MediaType.APPLICATION_JSON,
+                            examples = [
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2084]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 14]
```diff
@@ -1305,6 +2024,81 @@ class ProjectController extends ControllerBase{
...
+                                    @ExampleObject('''{
+    "key":"value",
+    "key2":"value2..."
+}'''
+                                    )
+                            ]
+                    ),
+                    @Content(
+                            mediaType = MediaType.TEXT_PLAIN,
+                            examples = [
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2094]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 15]
```diff
@@ -1361,6 +2155,65 @@ class ProjectController extends ControllerBase{
...
+                    @Tag(name="project")
+            ]
+    )
+    @ApiResponse(
+            responseCode = "200",
+            description = "Project details",
+            content = [
+                    @Content(
+                            mediaType = "application/text",
+                            examples = @ExampleObject('''key=value
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2197]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 16]
```diff
@@ -1361,6 +2155,65 @@ class ProjectController extends ControllerBase{
...
+key2=value''')
+                    ),
+                    @Content(
+                            mediaType = "application/json",
+                            examples = @ExampleObject('''{
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2202]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 17]
```diff
@@ -1426,6 +2279,90 @@ class ProjectController extends ControllerBase{
...
+            description = """Replace an individual configuration data with the submitted value.
+The response, based on `Accept` header, can be returned in the Text, XML or Json format.
+
+Authorization required: `configure` access for `project` resource type or `admin` or `app_admin` access for `user` resource type.""",
+            requestBody = @RequestBody(
+                    content = [
+                            @Content(
+                                    mediaType = MediaType.APPLICATION_JSON,
+                                    examples = [
+                                            @ExampleObject(
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2295]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 18]
```diff
@@ -1426,6 +2279,90 @@ class ProjectController extends ControllerBase{
...
+                                                    name = 'set-project-config',
+                                                    summary = "Replace an individual config settings",
+                                                    value = '''{ "[KEY]" : "key value" }'''
+                                            )
+                                    ]
+                            ),
+                            @Content(
+                                    mediaType = MediaType.TEXT_PLAIN,
+                                    examples = [
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2304]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 19]
```diff
@@ -1426,6 +2279,90 @@ class ProjectController extends ControllerBase{
...
+                    @Content(
+                            mediaType = MediaType.APPLICATION_JSON,
+                            examples = [
+                                    @ExampleObject('''{ "[KEY]" : "key value" }'''
+                                    )
+                            ]
+                    ),
+                    @Content(
+                            mediaType = MediaType.TEXT_PLAIN,
+                            examples = [
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2351]: same missing `schema` attribute

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 20]
```diff
@@ -1602,7 +2640,145 @@ class ProjectController extends ControllerBase{
...
+            ]
+    )
+    @ApiResponse(
+            responseCode = "200",
+            description = "Returns the Token to retrieve export status and download zip achive",
+            content = [
+                    @Content(
+                            mediaType = MediaType.APPLICATION_JSON,
+                            examples = [
+                                    @ExampleObject('''{
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2703]: same missing schema attribute

```suggestion
                            mediaType = MediaType.APPLICATION_JSON,
                            schema=@Schema(type='object'),
                            examples = [
                                    @ExampleObject('''{
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 21]
```diff
@@ -1602,7 +2640,145 @@ class ProjectController extends ControllerBase{
...
+                    @Tag(name="project")
+            ]
+    )
+    @ApiResponse(
+            responseCode = "200",
+            description = "Returns the Token to retrieve export status and download zip achive",
+            content = [
+                    @Content(
+                            mediaType = MediaType.APPLICATION_JSON,
+                            examples = [
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2763]:
```suggestion
                            mediaType = MediaType.APPLICATION_JSON,
                            examples = [
```
same missing schema attribute

```suggestion
                            mediaType = MediaType.APPLICATION_JSON,
                            schema=@Schema(type='object'),
                            examples = [
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 22]
```diff
@@ -1676,6 +2897,103 @@ class ProjectController extends ControllerBase{
...
+            responseCode = "200",
+            description = """Note: the import status indicates "failed" if any Jobs had failures, otherwise it indicates "successful" even if other files in the archive were not imported.
+
+Response will indicate whether the imported contents had any errors.
+
+Note: `other_errors` included since API v35""",
+            content = [
+                    @Content(
+                            mediaType = MediaType.APPLICATION_JSON,
+                            examples = [
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2965]: same missing schema attribute
```suggestion
                            mediaType = MediaType.APPLICATION_JSON,
                            schema=@Schema(type='object'),
                            examples = [
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy (hunk 19 above)
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2351]: same missing schema attribute
```suggestion
                            mediaType = MediaType.TEXT_PLAIN,
                            schema=@Schema(type='string'),
                            examples = [
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy (hunk 18 above)
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2304]: same missing schema attribute
```suggestion
                                    mediaType = MediaType.TEXT_PLAIN,
                                    schema=@Schema(type='string'),
                                    examples = [
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy (hunk 15 above)
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2197]: same missing schema attribute
```suggestion
                            mediaType = "application/text",
                            schema=@Schema(type='object'),
                            examples = @ExampleObject('''key=value
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy (hunk 14 above)
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2094]: same missing schema attribute
```suggestion
                            mediaType = MediaType.TEXT_PLAIN,
                            schema=@Schema(type='string'),
                            examples = [
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy (hunk 13 above)
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2084]: same missing schema attribute
```suggestion
                            mediaType = MediaType.APPLICATION_JSON,
                            schema=@Schema(type='object'),
                            examples = [
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 23]
```diff
@@ -1305,6 +2024,81 @@ class ProjectController extends ControllerBase{
...
+                                                    value = '''{
+    "key":"value",
+    "key2":"value2..."
+}'''
+                                            )
+                                    ]
+                            ),
+                            @Content(
+                                    mediaType = MediaType.TEXT_PLAIN,
+                                    examples = [
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2053]: same missing schema attribute
```suggestion
                                    mediaType = MediaType.TEXT_PLAIN,
                                    schema=@Schema(type='string'),
                                    examples = [
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy (hunk 3 above)
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1021]: same missing schema attribute
```suggestion
                            mediaType = "application/text",
                            schema=@Schema(type='string'),
                            examples = @ExampleObject('''key=value
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy [hunk 24]
```diff
@@ -1112,6 +1194,64 @@ class ScheduledExecutionController  extends ControllerBase{
...
+
+Since: V14''',
+        tags = ['jobs'],
+
+        parameters = @Parameter(
+            name = "id",
+            description = "Job ID",
+            in = ParameterIn.PATH,
+            required = true,
+            content = @Content(schema = @Schema(implementation = String))
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy:1213]: why not use `schema = @Schema(type = 'string')` as other cases  (`autorest` complains about this too)

File: rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy [hunk 25]
```diff
@@ -1112,6 +1194,64 @@ class ScheduledExecutionController  extends ControllerBase{
...
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy:1243]: why not use `schema = @Schema(type = 'string')` as other cases

File: rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy [hunk 26]
```diff
@@ -1164,6 +1304,64 @@ class ScheduledExecutionController  extends ControllerBase{
...
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy:1323]: why not use `schema = @Schema(type = 'string')` as other cases

File: rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy [hunk 27]
```diff
@@ -3164,34 +3829,72 @@ class ScheduledExecutionController  extends ControllerBase{
...
+                description = "Job ID",
+                in = ParameterIn.PATH,
+                required = true,
+                content = @Content(schema = @Schema(implementation = String))
+            ),
+            @Parameter(
+                name = "format",
+                description = '''can be "yaml" or "json" (API v44+) to specify the output format''',
+                in = ParameterIn.QUERY,
+                content = @Content(schema = @Schema(implementation = String))
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy:3897]: why not use `schema = @Schema(type = 'string')` as other cases

File: rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy [hunk 28]
```diff
@@ -3645,6 +4678,42 @@ class ScheduledExecutionController  extends ControllerBase{
...
+        method = "GET",
+        summary = 'Get Info About an Uploaded File',
+        description = '''Get info about an uploaded file given its ID.''',
+        tags = ['jobs'],
+        parameters = @Parameter(
+            name = "id",
+            description = "File ID",
+            in = ParameterIn.PATH,
+            required = true,
+            content = @Content(schema = @Schema(implementation = String))
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy:4692]: why not use `schema = @Schema(type = 'string')` as other cases

File: rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy [hunk 29]
```diff
@@ -3740,6 +4809,46 @@ class ScheduledExecutionController  extends ControllerBase{
...
+
+Authorization required: `delete` for the job.''',
+        tags = ['jobs'],
+        parameters = [
+            @Parameter(
+                name = "id",
+                description = "Job ID",
+                in = ParameterIn.PATH,
+                required = true,
+                content = @Content(schema = @Schema(implementation = String))
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy:4827]: why not use `schema = @Schema(type = 'string')` as other cases

**Review by ltamaster — COMMENTED** [review]
I add some comments based on the result of running the `autorest` client (that we use to generate `go-rundeck`) against the file `rundeck-api.yml`

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 30]
```diff
@@ -874,9 +1064,368 @@ class ProjectController extends ControllerBase{
...
+                    @Tag(name="project")
+            ]
+    )
+    @ApiResponse(
+            responseCode = "200",
+            description = "Project details",
+            content = [
+                    @Content(
+                            mediaType = "application/text",
+                            examples = @ExampleObject('''description: "my policy"
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1110]: same missing schema attribute

```suggestion
                            mediaType = "application/text",
                            schema=@Schema(type='string'),
                            examples = @ExampleObject('''description: "my policy"
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy (hunk 6 above)
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1121]: same missing schema attribute

```suggestion
                            mediaType = "application/yaml",
                            schema=@Schema(type='object'),
                            examples = @ExampleObject('''description: "my policy"
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy (hunk 7 above)
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1174]: same missing schema attribute
```suggestion
                                    mediaType = "application/yaml",
                                    schema=@Schema(type='object'),
                                    examples = @ExampleObject('''description: "my policy"
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy (hunk 8 above)
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1185]: same missing schema attribute

```suggestion
                                    mediaType = "application/json",
                                    schema=@Schema(type='object'),
                                    examples = @ExampleObject('''{
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 31]
```diff
@@ -1144,6 +1693,77 @@ class ProjectController extends ControllerBase{
...
+            summary = "To create or modify the `readme.md` and `motd.md` contents",
+            description = """Create or modify the `readme.md` and `motd.md` files, which are Markdown formatted and displayed in the Project listing page.
+The response, based on `Accept` header, can be returned in the Text, XML or Json format.
+
+Authorization required: `configure` access for `project` resource type or `admin` or `app_admin` access for `user` resource type.""",
+            requestBody = @RequestBody(
+                    content = [
+                            @Content(
+                                    mediaType = MediaType.APPLICATION_JSON,
+                                    examples = [
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1709]: same missing schema attribute

```suggestion
                                    mediaType = MediaType.APPLICATION_JSON,
                                    schema=@Schema(type='object'),
                                    examples = [
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 32]
```diff
@@ -1144,6 +1693,77 @@ class ProjectController extends ControllerBase{
...
+                                            @ExampleObject('''{"contents":"The readme contents"}''')
+                                    ]
+                            ),
+                            @Content(
+                                    mediaType = MediaType.TEXT_PLAIN,
+                                    examples = [
```
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1715]: same missing schema attribute
```suggestion
                                    mediaType = MediaType.TEXT_PLAIN,
                                    schema=@Schema(type='string'),
                                    examples = [
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy (hunk 9 above)
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1751]: same missing schema attribute
```suggestion
                            mediaType = "application/text",
                            schema=@Schema(type='string'),
                            examples = @ExampleObject('''The readme contents''')
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy (hunk 10 above)
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1755]: same missing schema attribute
```suggestion
                            mediaType = "application/json",
                            schema=@Schema(type='object'),
                            examples = @ExampleObject('''{"contents":"The readme contents"}''')
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy (hunk 11 above)
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1899]: same missing schema attribute
```suggestion
                            mediaType = "application/text",
                            schema=@Schema(type='string'),
                            examples = @ExampleObject('''The readme contents''')
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy (hunk 12 above)
Comment (ltamaster) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:1903]: same missing schema attribute

```suggestion
                            mediaType = "application/json",
                            schema=@Schema(type='object'),
                            examples = @ExampleObject('''{"contents":"The readme contents"}''')
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy [hunk 33]
```diff
@@ -1426,6 +2280,90 @@ class ProjectController extends ControllerBase{
...
+            method = "PUT",
+            summary = "Set the value.",
+            description = """Replace an individual configuration data with the submitted value.
+The response, based on `Accept` header, can be returned in the Text, XML or Json format.
+
+Authorization required: `configure` access for `project` resource type or `admin` or `app_admin` access for `user` resource type.""",
+            requestBody = @RequestBody(
+                    content = [
+                            @Content(
+                                    mediaType = MediaType.APPLICATION_JSON,
```
Comment (gschueler) [inline rundeckapp/grails-app/controllers/rundeck/controllers/ProjectController.groovy:2294]:
```suggestion
                                    mediaType = MediaType.APPLICATION_JSON,	                            
                                    schema=@Schema(type='object'),
```

File: rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy (hunk 24 above)
Comment (gschueler) [reply rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy:1213]: i believe it is the same result in the yaml, is it not the case? in the result, i see:
```yaml
      - name: id
        in: path
        description: Job ID
        required: true
        content:
          application/json:
            schema:
              type: string
```

Comment (ltamaster) [reply rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy:1213]: I got issues with `autorest`, but I think is a `autorest` issue

**Review by ltamaster — APPROVED** [review]

**Pull request merged**